tests/utils/constants.ts

```diff
@@ -13,6 +13,16 @@ export const AB_TEST_SCHEMA: AbTestSchema = {
   testWeight: '0.5'
 };
 
+export const PLACEHOLDERS: { [key: string]: string } = {
+  PLACEHOLDER_1: 'foo',
```

Comment:
The fact that these aren't cat names bums me out, but the test is nice nonetheless